prov/efa/src/rdm/rxr_tp.h

```diff
@@ -1,5 +1,5 @@
-#ifndef _EFA_TP_H_
-#define _EFA_TP_H_
+#ifndef _RXR_TP_H_
+#define _RXR_TP_H_
```

Comment:
Not sure if this was intended originally, but using `_EFA_TP_H_` as the include guard makes `efa_tp.h` and `rdm/rxr_tp.h` mutually exclusive.